src/domain/findWinners.js

```diff
@@ -0,0 +1,29 @@
+import { displayWinner } from "../view/output";
+// [
+//   {
+//     name: "A",
+//     count: 2
+//   },
+//   {
+//     name: "B",
+//     count: 1
+//   },
+//   {
+//     name: "C",
+//     count: 3
+//   }
+// ];
+function findWinners(cars) {
+  const maxAdvanceCount = Math.max(...cars.map((cars) => cars.count));
+  const winners = cars.filter((car) => car.count === maxAdvanceCount);
+
+  let winnersName = [];
+
+  for (let winner of winners) {
+    winnersName.push(winner.name);
+  }
+
+  return winnersName;
```

Comment:
winnersName 보다는 winnerNames 가 이름 문자열의 배열을 해당 변수가 담고 있음을 예측하기 쉬워보입니다.

또한 winners 나 winnerNames 는 굳이 정의가 필요하지 않은 변수이기 때문에 다음과 같이 코드를 축약해볼 수 있습니다.

```suggestion
  return cars.filter((car) => car.count === maxAdvanceCount).map((car) => car.name);
```